src/main/scala/encry/api/http/routes/WalletInfoApiRoute.scala

```diff
@@ -1,53 +1,247 @@
+import encry.modifiers.mempool.TransactionFactory
 import encry.settings.RESTApiSettings
 import encry.view.NodeViewHolder.ReceivableMessages.GetDataFromCurrentView
 import encry.view.history.History
+import encry.view.mempool.MemoryPool.NewTransaction
 import encry.view.state.UtxoState
 import encry.view.wallet.EncryWallet
 import io.circe.syntax._
-
+import org.encryfoundation.common.crypto.PrivateKey25519
+import org.encryfoundation.common.modifiers.mempool.transaction.{PubKeyLockedContract, Transaction}
+import org.encryfoundation.common.modifiers.state.box.{AssetBox, MonetaryBox, TokenIssuingBox}
+import org.encryfoundation.common.utils.Algos
+import org.encryfoundation.common.utils.TaggedTypes.ADKey
 import scala.concurrent.Future
-import scala.util.Random
+import scala.util.{Failure, Success, Try}
 
 case class WalletInfoApiRoute(dataHolder: ActorRef,
-                              restApiSettings: RESTApiSettings,
+                              settings: RESTApiSettings,
                               intrinsicTokenId: String)(implicit val context: ActorRefFactory)
   extends EncryBaseApiRoute with FailFastCirceSupport with StrictLogging {
 
-  override val route: Route = pathPrefix("wallet") { infoR ~ getUtxosR }
+  implicit val eqOptBytes = new Eq[Option[Array[Byte]]] {
+    override def eqv(x: Option[Array[Byte]], y: Option[Array[Byte]]) = {
+      x.exists(l => y.exists(_.sameElements(l)))
+    }
+  }
 
-  override val settings: RESTApiSettings = restApiSettings
+  override val route: Route = pathPrefix("wallet") {
+    infoR ~
+      getUtxosR ~
+      printAddressR ~
+      createKeyR ~
+      printPubKeysR ~
+      getBalanceR ~
+      transferR ~
+      transferContractR ~
+      createTokenR ~
+      dataTransactionR
+  }
 
-  private def getWallet: Future[EncryWallet] = (nodeViewHolder ?
-    GetDataFromCurrentView[History, UtxoState, EncryWallet, EncryWallet](_.vault))
-    .mapTo[EncryWallet]
+  private def getWallet: Future[EncryWallet] =
+    (dataHolder ?
+      GetDataFromCurrentView[History, UtxoState, EncryWallet, EncryWallet](_.vault))
+      .mapTo[EncryWallet]
 
   def infoR: Route = (path("info") & get) {
-    getWallet
-      .map { w =>
-        Map(
-          "balances" -> w.getBalances.map{ i =>
-            if (i._1._2 != intrinsicTokenId)
-              s"TokenID(${i._1._2}) for contractHash ${i._1._1} : ${i._2}"
-            else
-              s"TokenID(${i._1._2}) for contractHash ${i._1._1} : ${BigDecimal(i._2) / 100000000}"
-          }.asJson,
-          "utxosQty" -> Random.shuffle(w.walletStorage.getAllBoxes(1000)).length.asJson
-        ).asJson
+    getWallet.map { w =>
+      Map(
+        "balances" -> w.getBalances.map{ i =>
+          if (i._1._2 != intrinsicTokenId)
+            s"TokenID(${i._1._2}) for contractHash ${i._1._1} : ${i._2}"
+          else
+            s"TokenID(${i._1._2}) for contractHash ${i._1._1} : ${BigDecimal(i._2) / 100000000}"
+        }.asJson,
```

Comment:
You can remove here and on 66' line` .asJson` function, import `import io.circe.generic.auto._` and everything will work ok